test/csuite/truncated_log/main.c

```diff
@@ -128,16 +129,22 @@ fill_db(void)
     WT_SESSION *session;
     uint32_t i, max_key, min_key, units, unused;
     char k[K_SIZE], v[V_SIZE];
+    const char *table_config;
     bool first;
 
+    if (use_columns)
+        table_config = "key_format=r,value_format=S";
+    else
+        table_config = "key_format=S,value_format=S";
+
```

Comment:
Any reason not to use ternary operators here? (No strong feelings, shrug, feel free to ignore.)

Reply:
I think it's easier to read with the parallel structure (especially if in the future it gathers more cases or more parameters), but no particularly strong reason.